.nvmrc

```diff
@@ -1 +1 @@
-v14
+v20
```

Comment:
Just as heads up to record here, ArgoCD-UI uses nodejs v21 (Current not LTS)
I think is ok to leave it at version v20 much improvement jump from v14

Reply:
LTS is generally recommended. Workflows is also on [v20](https://github.com/argoproj/argo-workflows/blob/a927379f761fbffe733857e7a23d69511fba90de/ui/package.json#L16).

As I wrote [in Slack](https://cloud-native.slack.com/archives/C020XM04CUW/p1708632758034819?thread_ts=1708631268.852329&cid=C020XM04CUW), it's a devtool for front-end in any case, so it does not impact downstream